docs/docs/components/service-mesh.md

```diff
@@ -14,6 +14,20 @@ all workloads which have a specified configuration. The service mesh container f
 sets up `iptables` rules based on its configuration and then starts
 [Envoy](https://www.envoyproxy.io/) for TLS origination and termination.
 
+## Service mesh startup enforcement
+Since Contrast doesn't yet enforce the order in which the containers are started
+(see [Limitations](../features-limitations.md)), we deny all incoming connections
+until the service mesh resolves this deny rule.
```

Comment:
I stumbled over "this deny rule", because the implicit definition a few words earlier was not apparent. Maybe

> we deny all incoming connections until the service mesh is fully configured.